cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/MDSStreamMetaStore.java

```diff
@@ -61,33 +58,28 @@
   private Transactional<StreamMds, MetadataStoreDataset> txnl;
 
   @Inject
-  public MDSStreamMetaStore(CConfiguration conf, final TransactionSystemClient txClient, DatasetFramework framework) {
+  public MDSStreamMetaStore(CConfiguration conf,
```

Comment:
while we're changing these constructors, can we check if we can change them to private to prevent direct creation?